builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,31 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered        metrics.Counter
+	buildsTriggeredFailed  metrics.LabeledCounter
+	buildsTriggeredSuccess metrics.Counter
+)
+
+func init() {
+	buildMetrics := metrics.NewNamespace("builder", "", nil)
+
+	buildsTriggered = buildMetrics.NewCounter("builds_triggered", "Number of triggered image builds")
+	buildsTriggeredFailed = buildMetrics.NewLabeledCounter("builds_triggered_failed", "Number of failed image builds", "reason")
```

Comment:
I used that to keep the same `prefix`, but it's ok for me to shrink it to `builds_failed`. I'll do it.